src/classes/CallableApi_TEST.cls

```diff
@@ -0,0 +1,115 @@
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Callable API 
+* @group-content ../../ApexDocContent/CallableAPI.htm
+* @description Tests specific to Callable API. 
+*/
+@IsTest
+
+public with sharing class CallableApi_TEST {
+
+    private static final String TDTM_TRIGGER_ACTION = 'TDTM.disableAllTriggers';
+    
+    /*******************************************************************************************************
+    * @description verify the TDTM Callable action returns an Object
+    ********************************************************************************************************/
+    @IsTest
+    public static void verifyCallableReturnsAnObject() {
+       
+        Test.startTest();
+
+        // Call the API to disable All TDTM Triggers
+        Callable npspApi = (Callable) Type.forName('CallableApi').newInstance();
+        Map<String, Object> triggerArgs = new Map<String, Object>();
+        Object result = npspApi.call(TDTM_TRIGGER_ACTION, triggerArgs);
+
+        Test.stopTest();
+
+        Map<String, Object> triggerResult = (Map<String, Object>)result;
+        System.assertNotEquals(null, triggerResult, 'No result was returned from call');
+        System.assertEquals(TDTM_TRIGGER_ACTION, String.valueOf(triggerResult.get('action')), 'Action was not returned');
+    }    
+
+
+    /*******************************************************************************************************
+    * @description Call the API with an invalid method format and make sure an exception is thrown
+    ********************************************************************************************************/
+    @IsTest
+    public static void callableApiThrowsExceptionWhenMethodFormatIsInvalid() {
+       
+        Test.startTest();
+
+        Callable npspApi = (Callable) Type.forName('CallableApi').newInstance();
+        Map<String, Object> triggerArgs = new Map<String, Object>();
+        Boolean methodExceptionThrown = false;
+
+        try {
+            Object result = npspApi.call('InvalidFormat', triggerArgs);
+
+        } catch (CallableApi.MalformedMethodInvocationException apiException) {
+            methodExceptionThrown = true;
```

Comment:
A simpler alternative to using a var to flag that the exception is thrown is to add the following in the `try` block above:
```System.assert(false, 'The exception was not thrown');```